.env.sample

```diff
@@ -1,3 +1,4 @@
 # See https://docs.karakeep.app/configuration for more information
 DATA_DIR=<path>
-NEXTAUTH_SECRET=<secret>
\ No newline at end of file
+NEXTAUTH_SECRET=<secret>
+CRAWLER_WAYBACK_FALLBACK=true
```

Comment:
I don't think this config should be compulsory so it shouldn't need to be declared here.

Just a mention in docs/docs/03-configuration.md should be good enough.